### ISSUE TITLE
web_ui and Tensorboard not working?

### ISSUE THREAD

**Issue by sukibean163** [BMW-InnovationLab/BMW-YOLOv3-Training-Automation#3]
hi, i have run the auto training, and the output is 
2019-12-26 16:48:50,532 Events Log  : INFO     Running web_ui on port 8190
2019-12-26 16:48:50,532 Events Log  : INFO     Running YOLO API on port 9911
2019-12-26 16:48:50,537 Events Log  : INFO     Running Tensorboard on port 6006

use `curl "http://127.0.0.1:9911/summary"` got {"success":true,"start_time":"201912261647","data":{"current_training_iteration":"1179","total_loss":"0.750510","average_loss_error":"0.788038","current_learning_rate":"0.001000","total_time":"1.202105","number_of_images":"24759","mAP":"0.990001"}}
howerver,   curl "http://127.0.0.1:6006" **and** curl "http://127.0.0.1:8190" results:
curl: (56) Recv failure: Connection reset by peer
what's the problem?

**Comment by roy6324** [comment]
Hello, 
can you please provide the full output of the training command?

and can you use curl with --trace-ascii dump.txt to get more information on the error? ( for the ones that are failing) 
for example: curl --trace-ascii dump.txt "http://127.0.0.1:6006", then please provide the content of dump.txt.

**Comment by LSQI15** [comment]
Is the **mAP** evaluated on the test set or the training set?
"use curl "http://127.0.0.1:9911/summary" got {"success":true,"start_time":"201912261647","data":{"current_training_iteration":"1179","total_loss":"0.750510","average_loss_error":"0.788038","current_learning_rate":"0.001000","total_time":"1.202105","number_of_images":"24759","**mAP**":"0.990001"}}"
